README.md

```diff
@@ -91,6 +91,14 @@ This event was mislabeled \#46 on Meetup.
 
 2015-11-24 | 18:30 | [Meetup Event](https://www.meetup.com/Berlin-Hack-and-Tell/events/226945214/) | [Photos](https://www.meetup.com/Berlin-Hack-and-Tell/photos/26573373/) | [@c-base](https://www.c-base.org)
 
+* Hack&Tell Timer Reborn by Uwe
```

Comment:
Yes, here it is: https://github.com/c-base/hackandtell-raspberry Raspberry-Pi-based count-down timer written in OpenFrameworks (C++)